content/src/content/jcr_root/apps/core/wcm/components/carousel/v2/carousel/carousel.html

```diff
@@ -0,0 +1,84 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div data-sly-use.carousel="com.adobe.cq.wcm.core.components.models.Carousel"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     class="cmp-carousel"
+     role="group"
+     aria-label="${carousel.accessibilityLabel}"
+     aria-roledescription="carousel"
+     data-cmp-is="carousel"
+     data-cmp-autoplay="${(wcmmode.edit || wcmmode.preview) ? '' : carousel.autoplay}"
+     data-cmp-delay="${carousel.delay}"
+     data-cmp-autopause-disabled="${carousel.autopauseDisabled}">
+    <sly data-sly-test.hasItems="${carousel.items && carousel.items.size > 0}"></sly>
+    <div data-sly-test="${hasItems && carousel.autoplay}"
```

Comment:
@jakublamprecht, I think I would rather:

• keep the actions together under one wrapper
• call the items wrapper `.cmp-carousel__items`
• and then remove the `.cmp-carousel__content` wrapper

So:
```html
<div class="cmp-carousel">
  <div class="cmp-carousel__actions"><!-- the pause/play and rotation actions --></div>
  <div class="cmp-carousel__items"><!-- the items --></div>
  <div class="cmp-carousel__indicators"><!-- the indicators --></div>
</div>
```

what do you think?